tools/registration/registratie.md

```diff
@@ -2,7 +2,7 @@
 
 Te gek! Vul het volgende formulier in en we zullen snel contact met je opnemen!
 
-<p style="display: none" id="error" class="error">Eén van de velden was niet goed ingevuld, sorry! Probeer het nog eens.</p>
+<p class="form-error" id="error" class="error">Eén van de velden was niet goed ingevuld, sorry! Probeer het nog eens.</p>
```

Comment:
The /tools/ directory basically is exempt from the normal files - they're standalone aside from the `common` directory.

As such, the `.form-error` and `.form-emphasis` classes should be added to `tools/registration/registration.css` instead.